en/guide/serialization.md

```diff
@@ -220,12 +212,13 @@ def message_checksum(msg):
 
 <!-- From https://github.com/mavlink/pymavlink/blob/master/generator/mavparse.py#L385 -->
 
-> **Note** This uses the same x25 checksum that is used at runtime. It calculates a CRC over the message name (such as “RAW_IMU”) followed by the type and name of each field, space separated. The order of the fields is the order they are sent over the wire. For arrays, the array length is also added.
+> **Note** This uses the same CRC-16/MCRF4XX checksum that is used at runtime.
+  It calculates a CRC over the message name (such as “RAW_IMU”) followed by the type and name of each field, space separated.
+  The order of the fields is the order they are sent over the wire. For arrays, the array length is also added.
 
 ## Checksum {#checksum}
 
-The packet format includes a 2-byte CRC to allow detection of message corruption.
-The checksum is the same as used in ITU X.25 and SAE AS-4 standards ([CRC-16-CCITT](https://en.wikipedia.org/wiki/Cyclic_redundancy_check#Polynomial_representations_of_cyclic_redundancy_checks)), documented in [SAE AS5669A](http://www.sae.org/servlets/productDetail?PROD_TYP=STD&PROD_CD=AS5669A). 
+The packet format includes a 2-byte CRC-16/MCRF4XX to allow detection of message corruption.
 See the MAVLink source code for [the documented C-implementation](https://github.com/mavlink/c_library_v2/blob/master/checksum.h).
```

Comment:
@julianoes Also link to ## Integrity Checks / Checksum in the other doc for description?